rest/auth_handlers.go

```diff
@@ -0,0 +1,57 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/postmanlabs/postman-insights-agent/cfg"
+)
+
+func baseAuthHandler(req *http.Request) error {
+	postmanAPIKey, postmanEnv := cfg.GetPostmanAPIKeyAndEnvironment()
+	postmanInsightsVerificationToken := cfg.GetPostmanInsightsVerificationToken()
+
+	if postmanAPIKey != "" {
+		// Set postman API key as header
+		req.Header.Set("x-api-key", postmanAPIKey)
+
+		// Set postman env header if it exists
+		if postmanEnv != "" {
+			req.Header.Set("x-postman-env", postmanEnv)
+		}
+	} else if postmanInsightsVerificationToken != "" {
+		// Set postman team verification token as header
+		req.Header.Set("postman-insights-verification-token ", postmanInsightsVerificationToken)
```

Comment:
I think the approach I suggested in my comment could also remove the need to have this case -- create an authHandler that only works with the verification token, pass that in to the FrontClient created by the `daemon run` command, and then it is explicit which auth method that client will be using.